server/app.py

```diff
@@ -1,16 +1,144 @@
 """
 flask server entry point
 """
+import json
+from flask.wrappers import Response
+from flask.globals import request, session
+import requests
+from werkzeug.exceptions import abort
+from werkzeug.utils import redirect
+from google.oauth2 import id_token
+from google_auth_oauthlib.flow import Flow
+import os, pathlib
+import google
+from connect_db import connect_db, insert_into_db
+import jwt
+from flask_cors import CORS
 import os
-
+from dotenv import load_dotenv
 from api import create_app
 
+load_dotenv()
 app = create_app()
+CORS(app)
+#app.config['Access-Control-Allow-Origin'] = '*'
+#app.config["Access-Control-Allow-Headers"]="Content-Type"
 app.app_context().push()
 
+# bypass http
+os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "1"
+app.secret_key = os.getenv("SECRET_KEY")
+GOOGLE_CLIENT_ID = os.getenv("GOOGLE_CLIENT_ID")
+client_secrets_file = os.path.join(pathlib.Path(__file__).parent, "client-secret.json")
+algorithm = os.getenv("ALGORITHM")
+BACKEND_URL=os.getenv("BACKEND_URL")
+FRONTEND_URL=os.getenv("FRONTEND_URL")
+
+#database connection
+connect_db()
```

Comment:
is this to connect to the mongodb? if so, you shouldn't need it. in the api/.__init__ file the connection is initialized